File: templates/Uwp/_comp/_shared/Test.UnitTests.Core.SampleData.TreeView.NUnit/Param_ProjectName.Core.Tests.NUnit/Tests_postaction.cs
//{[{
using System.Threading.Tasks;
using Param_RootNamespace.Core.Services;
//}]}

namespace Param_RootNamespace.Core.Tests.NUnit
{
    public class Tests
    {
        //^^
        //{[{

        // TODO WTS: Remove or update this once your app is using real data and not the SampleDataService.
        // This test serves only as a demonstration of testing functionality in the Core library.
        [Test]
        public async Task EnsureSampleDataServiceReturnsTreeViewDataAsync()
        {
            var actual = await SampleDataService.GetTreeViewDataAsync();

            Assert.AreNotEqual(0, actual.Count);
        }
        //}]}
    }
}

File: templates/Uwp/_comp/_shared/Test.UnitTests.Core.SampleData.TreeView.NUnit._VB/Param_ProjectName.Core.Tests.NUnit/Tests_postaction.vb
'{[{
Imports System.Linq
Imports System.Threading.Tasks
Imports Param_RootNamespace.Core.Services
'}]}

Public class Tests
    '^^
    '{[{

    ' TODO WTS: Remove or update this once your app is using real data and not the SampleDataService.
    ' This test serves only as a demonstration of testing functionality in the Core library.
    <Test>
    Public Async Function EnsureSampleDataServiceReturnsTreeViewDataAsync() As Task
        Dim actual = Await SampleDataService.GetTreeViewDataAsync()

        Assert.AreNotEqual(0, actual.Count())
    End Function
    '}]}
End Class
